APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+
+Restricted / earmarked funding sources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Much of the Project’s funding comes from specific grants from private foundations and government agencies. In these cases, the fundable activities are explicitly described in the proposal text. Such funds must be spent following the proposed plan and in accordance with any other conditions imposed by the funding agency. The Finance Committee should make these restrictions clear to the community so that they may submit funding requests aligned with the existing grants.
+
+Whom to pay
+^^^^^^^^^^^
+1. All else being equal, give preference to existing contributors.
+2. When looking outside of the current team (from here on we use the word "team" to include anyone in the Astropy community, including code contributors, mentors, workshop facilitators, outreach, or any other activity for the benefit of the project), consider this an opportunity to make the project team more diverse.
+3. Pay attention to the following when considering paying someone:
+    * the necessary skills to execute the tasks,
+    * communication and self-management skills,
+    * experience contributing to and working with open source projects,
+    * likelihood of long-term involvement with the Astropy community.
+
+It will likely depend on the particular activity as to whether there’s already a clear best candidate within the Astropy team, or if we must look for new people to get involved. This decision should be taken in accordance with documented Project governance procedures. If the position is not filled from within the project, it should be publicly advertised to give a broader group of people a chance to apply for it.
+
+Compensating an appropriate amount 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Process
++++++++
+In general (see next section), we put the funding level directly in the hands of the community and those making funding requests. We use a process where proposers suggest work and include a budget request which they think is required to achieve the suggested work. These suggestions are then put to the community for input and feedback. As part of the review process, the community is expected to take “value-for-money” into account. This mechanism places the trust to decide when compensation is unreasonably high or low to the community. In other cases, the Finance Committee will determine an appropriate amount taking into account the discussion and rationale below.
+
+Discussion and Rationale
+++++++++++++++++++++++++
+This is motivated by the fundamental tension that we, as a community, want to get the most out of the funds available, but we also want to avoid a “race to the bottom” where developers undercut others on price until the pay is too low to live on.
+Setting an appropriate level of pay for certain work is a difficult task, since a range of factors influences what people think is “fair”. This includes, to name just a few considerations,
```

Comment:
```suggestion
Setting an appropriate level of pay for certain work is a difficult task, since a range of factors influence what people think is “fair.” This includes, to name just a few considerations,
```